src/site/includes/detail-page.html

```diff
@@ -10,16 +10,16 @@
 {% endcomment %}
 
 <main class="va-l-detail-page">
-  {% if hidesidenav === empty %}
+  {% if hidesidenav == empty %}
```

Comment:
Strictly equals isn't a [thing](https://shopify.github.io/liquid/basics/operators/) in Liquid. Not sure how this was working before.